src/features/messaging/messaging-contacts-bridge.js

```
@@ -1,7 +1,5 @@
-import { contactsService } from '../features/contacts/index.js';
-import { messagingController } from '../features/messaging/messaging-controller.js';
-
-// TODO: Fit into standardized pattern - Use appBus for this, or standardize on this pattern with domain event busses
+import { contactsService } from '../contacts/index.js';
```

ESLint

[error] There is no rule allowing dependencies from elements of type "feature" and featureName "messaging" to elements of type "feature" and featureName "contacts"

(boundaries/dependencies)